crates/core/src/node/in_memory_ext.rs

```diff
@@ -321,7 +322,9 @@ impl InMemoryNode {
         Ok(())
     }
 
-    pub async fn remove_block_timestamp_interval(&self) -> Result<bool> {
+    pub async fn remove_block_timestamp_interval(
+        &self,
+    ) -> std::result::Result<bool, AnvilNodeError> {
```

Comment:
Thought for improvement: might makes sense to add `type Result<T> = std::result::Result<T, AnvilNodeError>;` inside `zksync_error`'s auto-generation

Reply:
Will do -- there is an [issue](https://github.com/matter-labs/zksync-error/issues/22) for that even :)